### PR TITLE
UBER-769 Implement backlinks in server plugin

### PR DESCRIPTION
**Pull request by aonnikov**
# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: dev/prod/package.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
     "@hcengineering/platform": "^0.6.9",
     "@hcengineering/ui": "^0.6.10",
     "@hcengineering/theme": "^0.6.3",
+    "@hcengineering/tiptap": "^0.6.0",
```
Comment (haiodo) [inline dev/prod/package.json]: Better to make it text, not a tiptap

File: server-plugins/chunter-resources/package.json [hunk 2]
```diff
@@ -36,6 +37,11 @@
     "@hcengineering/workbench": "^0.6.8",
     "@hcengineering/notification": "^0.6.14",
     "@hcengineering/server-notification": "^0.6.1",
-    "@hcengineering/server-notification-resources": "^0.6.0"
+    "@hcengineering/server-notification-resources": "^0.6.0",
+    "@hcengineering/server": "^0.6.0",
+    "@hcengineering/tiptap": "^0.6.0",
```
Comment (haiodo) [inline server-plugins/chunter-resources/package.json]: I expect only our text plugin to be here, please remove tiptap direct dependencies.

**Review by haiodo — APPROVED** [review]

File: server-plugins/chunter-resources/package.json (hunk 2 above)
Comment (aonnikov) [reply server-plugins/chunter-resources/package.json]: Removed tiptap dependencies

File: dev/prod/package.json (hunk 1 above)
Comment (aonnikov) [reply dev/prod/package.json]: Renamed

**Review by haiodo — APPROVED** [review]

**Pull request merged**